source/update/prebuilt.html.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Updating pre-built binaries
+---
+
```

Comment:
we should maybe mention somewhere that these prebuilt binaries are for linux amd64.